.github/workflows/gardening.yml

```diff
@@ -6,7 +6,8 @@ on:
     types: [opened, reopened]
   push:
     branches:
-      - master # Every time a PR is merged to master.
+      - trunk # Every time a PR is merged to trunk.
+      - master # Out-going alias for trunk.
```

Comment:
This is not sufficient. `projects/github-actions/repo-gardening/src/tasks/add-milestone/index.js` line 20 and `projects/github-actions/repo-gardening/src/tasks/wpcom-commit-reminder/index.js` line 80 also need updating for this to work right.